packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -440,13 +440,17 @@ export const transformers = (model: ModelData): string[] => {
 	}
 
 	if (model.pipeline_tag && LIBRARY_TASK_MAPPING.transformers?.includes(model.pipeline_tag)) {
-		const pipelineSnippet = [
-			"# Use a pipeline as a high-level helper",
-			"from transformers import pipeline",
-			"",
-			`pipe = pipeline("${model.pipeline_tag}", model="${model.id}"` + remote_code_snippet + ")",
-		].join("\n");
-		return [pipelineSnippet, autoSnippet];
+		const pipelineSnippet = ["# Use a pipeline as a high-level helper", "from transformers import pipeline", ""];
+
+		if (model.tags.includes("conversational")) {
+			pipelineSnippet.push("messages = [", '    {"role": "user", "content": "Who are you?"},', "]");
```

Comment:
With formatting this looks quite bad compared to what we had before hehe

Reply:
you can do it in two calls to push if you prefer

Reply:
It's fine like this I think, thanks!